core/src/main/scala/cats/ContravariantSemigroupal.scala

```diff
@@ -13,3 +13,5 @@ import simulacrum.typeclass
       def G = Functor[G]
     }
 }
+object ContravariantSemigroupal extends SemigroupalArityFunctions {
```

Comment:
nitpick, but can we get rid of these curly braces? :)

Reply:
Hah, yeah, for sure.